src/js/view/preview.js

```diff
@@ -6,7 +6,7 @@ const Preview = function(_model) {
 };
 
 function validState(state) {
-    return state === 'complete' || state === 'idle' || state === 'error';
+    return state === 'complete' || state === 'idle' || state === 'error' || state === 'buffering';
```

Comment:
Adding buffering here means we will load a poster while loading a video, which is what we were trying to avoid.